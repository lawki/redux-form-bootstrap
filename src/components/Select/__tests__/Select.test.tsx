import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
// import { Input } from 'reactstrap';
import { SelectForm as Select, SelectOption } from '..';

const options: SelectOption[] = [
    {
        label: '1',
        value: 1
    },
    {
        label: '2',
        value: 2
    }
]

describe('Select', () => {
    let wrapper: ShallowWrapper;
    beforeAll(() => {
        wrapper = shallow(<Select options={options} value={1}/>);
    });
    it('renders', () => {
        expect(wrapper).toMatchSnapshot();
    });
})