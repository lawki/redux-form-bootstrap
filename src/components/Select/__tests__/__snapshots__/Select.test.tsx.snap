// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select renders 1`] = `
<FormGroup
  tag="div"
>
  <Input
    onChange={[Function]}
    type="select"
    value={1}
  >
    <option
      value=""
    >
      Nothing is Selected
    </option>
    <option
      key="1"
      value={1}
    >
      1
    </option>
    <option
      key="2"
      value={2}
    >
      2
    </option>
  </Input>
</FormGroup>
`;
